src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewGuidCommand.cs

```diff
@@ -9,16 +11,44 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// The implementation of the "new-guid" cmdlet.
     /// </summary>
-    [Cmdlet(VerbsCommon.New, "Guid", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=2097130")]
+    [Cmdlet(VerbsCommon.New, "Guid", DefaultParameterSetName = "Default", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=2097130")]
     [OutputType(typeof(Guid))]
     public class NewGuidCommand : Cmdlet
     {
         /// <summary>
-        /// Returns a guid.
+        /// Generates an instance of the Guid structure whose value is all zeros.
+        /// </summary>
+        [Parameter(ParameterSetName = "Empty")]
+        public SwitchParameter Empty { get; set; }
+
+        /// <summary>
+        /// Converts a string to a Guid.
+        /// </summary>
+        [Parameter(ParameterSetName = "FromString")]
+        [ValidateNotNullOrWhiteSpace]
```

Comment:
are there arguments to be made to have this accept pipeline input?

Reply:
@JamesWTruher I could see a use case for pipeline input.

Reply:
Pipeline input would be nice, that or even a `ConvertTo-Guid` cmdlet.

Reply:
It now works with pipeline input